quipucords/api/reports/model.py

```diff
@@ -4,21 +4,20 @@
 
 from django.db import models
 
-from api.common.common_report import REPORT_TYPE_CHOICES, REPORT_TYPE_DETAILS
 from api.inspectresult.model import RawFactEncoder
 
 
-class DetailsReport(models.Model):
+class Report(models.Model):
     """A reported set of facts."""
 
-    report_type = models.CharField(
-        max_length=11, choices=REPORT_TYPE_CHOICES, default=REPORT_TYPE_DETAILS
-    )
     report_version = models.CharField(max_length=64, null=False)
+    # unique identifier required by yupana/insights
```

Comment:
Is this referring to `report_platform_id`?

Please add the name to the comment so that the purpose is crystal clear. 🔮

Reply:
yep, it its. will add it =D